modules/developers-guide/pages/cli-reference.adoc

```diff
@@ -77,6 +81,8 @@ Use the following subcommands to specify the operation you want to perform or to
 
 |`+help+` |Displays usage information for a specified subcommand.
 
+|`+identity+` |Manages the identities used to communicate with the Internet Computer network when accessing deployed canisters.
```

Comment:
> when accessing deployed canisters

We actually communicate with the Internet Computer before deploying (installing), for example when registering a new canister.

Reply:
Good point. This isn't obvious because it is the default identity until after you register a canister. So it's fair to assume that, for example, after I run dfx identity new registeration_user, I could run dfx --identiry registeration_user canister create --all, is that right?

Reply:
Yes, you could run those two commands in sequence

Reply:
cool